pybar_fei4_interpreter/Interpret.cpp

```diff
@@ -527,6 +530,12 @@ void Interpret::useTriggerTimeStamp(bool useTriggerTimeStamp)
 	_useTriggerTimeStamp = useTriggerTimeStamp;
 }
 
+void Interpret::setTriggerFormat(const char* rTriggerFormat)

```

Comment:
`const char*` is C-style code and not a modern way of writing C/C++ code. You should use a [std::string](http://www.cplusplus.com/reference/string/string/)